model/model.py

```diff
@@ -17,12 +17,6 @@
 class ClassificationModel(ClassificationModelBase):
     """
     Wrapper class for the model to be served.
-
-    Attributes:
-        classes: list, contains the labels
-        classes_n: int, number of classes
-        model_path: str, path to the model file
-        model: the object representing the model, to be loaded with _load_model()
```

Comment:
I would keep this here, since the developer won't know about these attributes without going into the `telesto-base` source code.

Reply:
This documentation breaks the docs guidelines since these attributes don't belong to this class but rather to its parent. Any reasonable IDE should show the fields and methods available on the parent class.